### PR TITLE
CSS to handle tab overflow

### PR DESCRIPTION
**Pull request by joshmadewell**
Fix for issue #50

### CODE REVIEW

**Comment by joshmadewell** [comment]
screen shot 2015-12-31 at 11 02 39 am

File: src/ui/less/components/tabView.less [hunk 1]
```diff
@@ -13,13 +13,16 @@
   text-align: center;
   color: @theme-text-color;
   list-style: none;
-  border-top: 1px solid transparent;
+  border: 1px solid black;
```
Comment (officert) [inline src/ui/less/components/tabView.less]: can we go back to transparent here? not sure if this was switched for debugging purposes, looks funky tho

**Comment by joshmadewell** [comment]
I changed the border back to transparent. It made sense before I added the fadeout :D

**Comment by officert** [comment]
:+1: thanks man, looks great!

**Pull request merged**